base/docs/helpdb/Base.jl

```diff
@@ -978,10 +978,12 @@ references in `r` are replaced with the corresponding matched text.
 replace
 
 """
-    randexp([rng], [dims...])
+    randexp([rng], [F], [dims...])
+
+Generate a random number of type `F` according to the exponential distribution with scale 1.
+The type `F` can be `Float16`, `Float32` or `Float64` and defaults to the latter.
```

Comment:
(serial) comma before "or" is a more common style.   "latter" is not grammatically correct in a list of three items.   Maybe "... can be `Float16`, `Float32`, or `Float64` (the default)" or even just "should be a floating-point type (defaulting to `Float64`)" to indicate the intended path for future extensions.

It is more common in the documentation to use `T` for a type parameter; also, in the near future I would like to add complex types, and it would be good at some point to call `mpfr_grandom` for `BigFloat`.